src/Controls/src/Build.Tasks/nuget/buildTransitive/netstandard2.0/maui.aotprofile.txt

```diff
@@ -3371,13 +3102,16 @@ Methods:
 	System.Reflection.CustomAttributeData[] System.Reflection.CustomAttribute:GetCustomAttributesDataInternal (System.Reflection.ICustomAttributeProvider)
 	System.Reflection.CustomAttributeData[] System.Reflection.CustomAttribute:GetPseudoCustomAttributesData (System.Reflection.ICustomAttributeProvider,System.Type)
 	System.Reflection.CustomAttributeData[] System.Reflection.CustomAttribute:GetPseudoCustomAttributesData (System.Type)
-	System.Reflection.CustomAttributeData[] System.Reflection.RuntimeParameterInfo:GetPseudoCustomAttributesData ()
 	System.Reflection.CustomAttributeNamedArgument[] System.Reflection.RuntimeCustomAttributeData:UnboxValues (object[])
 	System.Reflection.CustomAttributeTypedArgument System.Array:InternalArray__get_Item (int)
 	System.Reflection.CustomAttributeTypedArgument System.Collections.ObjectModel.ReadOnlyCollection`1<System.Reflection.CustomAttributeTypedArgument>:System.Collections.Generic.IList<T>.get_Item (int)
 	System.Reflection.CustomAttributeTypedArgument[] System.Reflection.RuntimeCustomAttributeData:UnboxValues (object[])
-	System.Reflection.FieldAttributes System.Reflection.RuntimeFieldInfo:get_Attributes ()
-	System.Reflection.FieldInfo System.Net.CookieParser:get_IsQuotedDomainField ()
+	System.Reflection.Emit.ILGenerator System.Reflection.Emit.DynamicMethod:GetILGenerator (int)
+	System.Reflection.Emit.Label System.Reflection.Emit.RuntimeILGenerator:BeginExceptionBlock ()
+	System.Reflection.Emit.Label System.Reflection.Emit.RuntimeILGenerator:DefineLabel ()
+	System.Reflection.Emit.LocalBuilder System.Reflection.Emit.ILGenerator:DeclareLocal (System.Type)
+	System.Reflection.Emit.LocalBuilder System.Reflection.Emit.RuntimeILGenerator:DeclareLocal (System.Type,bool)
+	System.Reflection.Emit.RuntimeILGenerator System.Reflection.Emit.DynamicMethod:GetILGeneratorInternal (int)
```

Comment:
These are new; going to figure out where they are coming from.

Reply:
I'm going to try to reproduce this in a `dotnet new android` app, but what happens is these end up in the profile:

* https://github.com/dotnet/runtime/blob/819e288da8689a5aefe9f201865b0f44f00537d4/src/libraries/Microsoft.Extensions.DependencyInjection/src/ServiceLookup/ILEmit/ILEmitResolverBuilder.cs#L63
* https://github.com/dotnet/runtime/blob/819e288da8689a5aefe9f201865b0f44f00537d4/src/libraries/Microsoft.Extensions.DependencyInjection/src/ServiceLookup/ILEmit/ILEmitResolverBuilder.cs#L351

But they are never called when I look at `dotnet-trace`.

What's even weirder though, is the runtime loads the code for them:
```
08-23 16:30:54.710  3690  3752 D Mono    : AOT: FOUND method Microsoft.Extensions.DependencyInjection.ServiceLookup.ILEmitResolverBuilder:GenerateMethodBody (Microsoft.Extensions.DependencyInjection.ServiceLookup.ServiceCallSite,System.Reflection.Emit.ILGenerator) [0x74cb72f3c0 - 0x74cb7301d0 0x74cb73701f]
...
08-23 16:30:54.712  3690  3752 D Mono    : AOT: FOUND method System.Reflection.Emit.RuntimeILGenerator:BeginExceptionBlock () [0x74d4632670 - 0x74d4632950 0x74d46a4c54
```

So, the usage of a delegate, seems like it causes the runtime to "preload" AOT images for code that is never run? I think a smaller repro might help explain what is going on.